core-db/src/main/java/io/novafoundation/nova/core_db/AppDatabase.kt

```diff
@@ -254,6 +255,7 @@ abstract class AppDatabase : RoomDatabase() {
                     .addMigrations(AddLocalMigratorVersionToChainRuntimes_57_58, AddGloballyUniqueIdToMetaAccounts_58_59)
                     .addMigrations(ChainNetworkManagement_59_60, AddBalanceHolds_60_61, ChainNetworkManagement_61_62)
                     .addMigrations(TinderGovBasket_62_63, AddChainForeignKeyForProxy_63_64, AddBrowserTabs_64_65)
+                    .addMigrations(AddBrowserTabs_65_66)
```

Comment:
I think we can merge it with 64 -> 65 since we havent released 64 -> 65 yet

Reply:
We discussed this once. I don't think there's any point in wasting time on this